src/seedu/addressbook/parser/Parser.java

```diff
@@ -116,7 +116,8 @@ public Command parseCommand(String userInput) {
                 return prepareDelete(arguments);
 
             case EditCommand.COMMAND_WORD:
-                return processEdit(arguments);
+//                return processEdit(arguments);
```

Comment:
Don't comment out unused code. Just delete it. You can restore it by reversing/referencing the commit.